### PR TITLE
WSTEAMA-1458 - Setup static assets for 2nd preview environment

### PR DESCRIPTION
**Pull request by louisearchibald**
Resolves JIRA [[1458](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1458)]

Overall changes
======
Sets up static assets in Simorgh for a secondary preview environment

Code changes
======

- Renames existing `preview.env` file to `preview1.env`.
- Amends `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` to reflect this name change.
- Updates existing command `build:preview` to be `build:preview1` & updates the path to .env file in the script body.
- Creation of a `preview2.env` with the `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` value pointing to this new file.
- Creation of a `build:preview2` command with a path in the script body pointing to the file created above.

Testing
======
Branch was deployed to the preview environment and then tested. The environment was not broken by any of the changes, including the change made in [this infrastructure PR](https://github.com/bbc/simorgh-infrastructure/pull/1863). The network tab was inspected for any unexpected 404s (none present), and the new static asset for `preview1` is now shown in the header.

Screenshot 2024-12-06 at 11 27 24

### CODE REVIEW

File: envConfig/preview1.env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## NB only variables prefixed with SIMORGH_ are available on both the server and client
 SIMORGH_BASE_URL=https://www.test.bbc.com
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://static.test.files.bbci.co.uk
-SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh-preview-assets/public/
+SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh-preview1-assets/public/
```
Comment (karinathomasbbc) [inline envConfig/preview1.env]: Would it be better to keep the name in line with the new simorgh preview name?
```suggestion
SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh1-preview-assets/public/
```

File: envConfig/preview2.env [hunk 2]
```diff
@@ -0,0 +1,32 @@
+## NB only variables prefixed with SIMORGH_ are available on both the server and client
+SIMORGH_BASE_URL=https://www.test.bbc.com
+SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://static.test.files.bbci.co.uk
+SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh-preview2-assets/public/
```
Comment (karinathomasbbc) [inline envConfig/preview2.env]:
```suggestion
SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh2-preview-assets/public/
```

File: envConfig/preview1.env (hunk 1 above)
Comment (louisearchibald) [reply envConfig/preview1.env]: Thanks, changed now for both

**Review by pvaliani — APPROVED** [review]
LGTM

**Review by karinathomasbbc — APPROVED** [review]
Thanks for making the changes!

File: package.json [hunk 3]
```diff
@@ -32,7 +32,8 @@
     "build:profile": "rm -rf build && cp envConfig/local.env .env && IS_PROD_PROFILE=true NODE_ENV=production webpack",
     "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
     "build:live:debug": "rm -rf build && awk '{sub(/LOG_DIR=.+/,\"LOG_DIR='log'\")}1' envConfig/live.env > .env && NODE_ENV=production webpack",
-    "build:preview": "cp envConfig/preview.env .env && NODE_ENV=production webpack",
+    "build:preview1": "cp envConfig/preview1.env .env && NODE_ENV=production webpack",
```
Comment (andrewscfc) [inline package.json:36]:
```suggestion
    "build:preview": "cp envConfig/preview1.env .env && NODE_ENV=production webpack",
    "build:preview1": "cp envConfig/preview1.env .env && NODE_ENV=production webpack",
```
As in https://github.com/bbc/simorgh-infrastructure/pull/1863#discussion_r1871493111 we need to maintain backward compatibility as `TARGET_ENV` is [not configurable](https://github.com/bbc/simorgh-infrastructure/blob/651675c84c9e17a8d78793a16b4a7b0f45dd0846/packages/preview-deploy/lib/preview-deploy-stack.js#L84) yet. 

I appreciate this wasn't in the ticket, this detail was missed sorry

**Review by andrewscfc — CHANGES_REQUESTED** [review]

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**